front/app/containers/CustomPageShow/HeaderContent.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import useLocalize from 'hooks/useLocalize';
+import {
+  Container,
+  HeaderTitle,
+  HeaderSubtitle,
+} from 'containers/LandingPage/SignedOutHeader/HeaderContent';
+import { InjectedIntlProps } from 'react-intl';
+import { injectIntl } from 'utils/cl-intl';
+import { Multiloc } from 'typings';
+
+type TAlign = 'center' | 'left';
+interface Props {
+  fontColors: 'light' | 'dark';
+  align?: TAlign;
+  headerMultiloc: Multiloc;
+  subheaderMultiloc: Multiloc;
+  hasHeaderBannerImage: boolean;
```

Comment:
Would just pass the slug/id of the page, then use useCustomPage. No need to pass these as props then. The same goes for other components.